### PR TITLE
Creates a success page for after a destination is created

### PR DESCRIPTION
**Pull request by rysimone**

### CODE REVIEW

**Comment by rysimone** [comment]
image

File: src/main/webapp/destinationCreation.js [hunk 1]
```diff
@@ -123,7 +123,7 @@ function removeElements(elemcls) {
  * @param {String} data placeId to be checked if it is already in datastore
  */
 function searchForDuplicate(data) {
-  fetch('/duplicates?' + PLACE_ID_URL_PARAM + '=' + data)
+  fetch('/duplicates?' + PLACE_ID_URL_PARAM + '=' + data, {method: 'POST'})
```
Comment (tjgoog) [inline src/main/webapp/destinationCreation.js:126]: Shouldn't this still be GET? GET is the default.

File: src/main/webapp/destinationCreationSuccess.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    <script src="success.js"></script>
+  </head>
+  <body class="background" onload="confirmCreation()">
+    <div class="content">
+      <div id="confirmation-block">
+        <h1>Destination Successfully Created!</h1>
+      </div>
+      <div class="button-row">
+        <a href="index.html" class="button">Home Page</a>
+        <a href="generateHunt.html" class="button">Generate a Hunt</a>
```
Comment (megankj) [inline src/main/webapp/destinationCreationSuccess.html:19]: maybe you should have another button to add another destination?

Comment (rysimone) [reply src/main/webapp/destinationCreationSuccess.html:20]: Yea that's not a bad idea

**Pull request merged**